addons/point_of_sale/static/src/xml/Screens/OrderManagementScreen/OrderlineDetails.xml

```diff
@@ -15,6 +15,12 @@
                 </strong>
                 <span><t t-esc="pricePerUnit" /></span>
             </li>
+            <t t-if="props.line.get_customer_note()">
```

Comment:
add a getter 'customerNote' in "OrderlineDetails." and use it here instead of props.line.get_customer_note

Reply:
This make sense, I've done the same for the `Orderline` component